wiki/People/Nomination_Assessment_Team/en.md

```diff
@@ -181,18 +179,19 @@ The following NAT members are exclusively responsible for structural tasks, and
 | :-- | :-- |
 | ::{ flag=GB }:: [-Mo-](https://osu.ppy.sh/users/2202163) | Handling internal discussions, affairs, and documentation |
 | ::{ flag=SE }:: [Naxess](https://osu.ppy.sh/users/8129817) | [Aiess](https://github.com/Naxesss/Aiess) and [Mapset Verifier](https://github.com/Naxesss/MapsetVerifier) maintenance |
-| ::{ flag=US }:: [pishifat](https://osu.ppy.sh/users/3178418) | [NAT/BN website](https://bn.mappersguild.com) development |
+| ::{ flag=US }:: [pishifat](https://osu.ppy.sh/users/3178418) | [BN Management website](https://bn.mappersguild.com) development |
 | ::{ flag=US }:: [radar](https://osu.ppy.sh/users/7131099) | NAT leadership |
 
 The following NAT members are primarily evaluators, but also contribute to structural tasks:
 
 | User | Task |
 | :-- | :-- |
+| ::{ flag=DE }:: [Capu](https://osu.ppy.sh/users/2474015) | handling osu!taiko [SEV ratings](/wiki/People/Nomination_Assessment_Team/SEV_rating) |
 | ::{ flag=BR }:: [Dada](https://osu.ppy.sh/users/9119507) | BN Mentorship organisation |
```

Comment:
```suggestion
| ::{ flag=BR }:: [Dada](https://osu.ppy.sh/users/9119507) | BN mentorship organisation |
```

Reply:
intended to be a proper noun, aka the name of the project